tests/Integration/Database/DatabaseTransactionsTest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Illuminate\Tests\Integration\Database;
+
+use Illuminate\Support\Facades\DB;
+
+class DatabaseTransactionsTest extends DatabaseTestCase
```

Comment:
Change to `DatabaseTestCase` so this will be executed on MySQL, Postgres and MsSQL too

Reply:
Good catch. Thank you @crynobone!